Userland/Libraries/LibCompress/AsyncDeflate.cpp

```diff
@@ -248,7 +248,7 @@ Coroutine<ErrorOr<void>> decode_codes(AsyncInputLittleEndianBitStream& stream, C
 }
 }
 
-DeflateDecompressor::DeflateDecompressor(NonnullOwnPtr<AsyncInputStream>&& input)
+DeflateDecompressor::DeflateDecompressor(MaybeOwned<AsyncInputStream>&& input)
```

Comment:
Looks to have ended up in the wrong commit (same for the header file).

Reply:
Well, this was deliberate: non-owning inflate is not particularly useful by itself and is only used in zlib decompression. I can fix up this change, though, if you wish